Shared/Extensions/String.swift

```diff
@@ -56,5 +56,12 @@ extension String {
         
         return mut
     }
+    
+    public func pluralize(_ count: Int, pluralization : String = "s") -> String {
```

Comment:
I think the second argument should be a nillable string to be a custom replacement and if there is no argument or nil it appends an s. This should allow us to cover irregular pluralizations